spec/index.html

```diff
@@ -2042,7 +2061,7 @@ <h2>Acknowledgments</h2>
   <h2>Substantive changes since RDF 1.1</h2>
 
   <ul>
-    <li> None so far. </li>
+    <li> The RDF entailment rule <a>rdfD1a</a> was added.  This rule was not added for RDF entailment when the two built-in dataypes were added to RDF entailment. </li>
```

Comment:
I find these sentences confusing. Both use the past tense, but I'm assuming that they are referring to different points in the past. More precisely, I'm assuming (without being certain) that they mean: "The [rule] was added *(in 2023)*. This rule was not added *(in 2014)*...". In any casen this needs clarification.

I would suggest replacing, in the 2nd sentence, "was not added" by "had not been added", but I defer to native speakers to find the best wording.

Reply:
The two built-in datatypes of RDF 1.1's RDF entailment are `xsd:string` and `rdf:langString`.
```suggestion
    <li> The RDF entailment rule <a>rdfD1a</a> was added in RDF 1.2.  This rule should have been included in RDF 1.1 when the two built-in datatypes (<code>xsd:string</code> and <code>rdf:langString</code>) were added to RDF entailment. </li>
```

Reply:
Just one tweak to AZ's update, to drop the initial `The`
```suggestion
    <li> RDF entailment rule <a>rdfD1a</a> was added in RDF 1.2.  This rule should have been included in RDF 1.1 when the two built-in datatypes (<code>xsd:string</code> and <code>rdf:langString</code>) were added to RDF entailment. </li>
```